File: 2/Product.py
class Product():
	__price: int
	__idProduct: str

	
	def __init__(self, price: int, idProduct: str):
		self.__price = price
		self.__idProduct = idProduct

	def setPrice(self, price):
		self.__price = price

	def getPrice(self):
		return self.__price

	def setIdProduct(self, idProduct):
		self.__idProduct = idProduct

	def getIdProduct(self):
		return self.__idProduct

	def printProduct(self):
		print("Harga : ", self.__price)
		print("ID Produk : ", self.__idProduct)

File: 2/Product.php
<?php declare(strict_types=1);

class Product
{
	private int $price;
	private string $idProduct;

	// construct
	public function __construct(int $price, string $idProduct)
	{
		$this->price = $price;
		$this->idProduct = $idProduct;
	}

	public function setPrice(int $price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->$price;
	}

	public function setIdProduct(string $idProduct)
	{
		$this->idProduct = $idProduct;
	}

	public function getIdProduct()
	{
		return $this->idProduct;
	}

	public function printProduct()
	{
		echo "Harga : " . $this->price . "<br>";
		echo "ID Produk : " . $this->idProduct . "<br>";
	}
}

?>